css/css-grid/grid-definition/support/testing-utils.js

```diff
@@ -19,7 +19,7 @@ var TestingUtils = (function() {
             grid.style.gridTemplateRows = rowsStyle;
             checkGridTemplateColumns(grid, columnsComputedValue);
             checkGridTemplateRows(grid, rowsComputedValue);
-        }, (label||'') + " '" + gridId + "' with: grid-template-columns: " + columnsStyle  + "; and grid-template-rows: " + rowsStyle + ";");
+        }, (label + " " || "") + "'" + gridId + "' with: grid-template-columns: " + columnsStyle  + "; and grid-template-rows: " + rowsStyle + ";");
```

Comment:
Should be like `label ? label + "" : ""`, since `undefined + " "` is a truthy string.